File: Assignment4/Assignment4.py
distance = int(input("Enter distance: "))
total=0
charge=0
if distance<=0:
   print("Invalid distance")
elif distance>0 and distance<2:
   charge=50
   total=total+charge
elif distance>=2 and distance<5:
   charge=20;	
   total=total+charge*distance
elif distance>=5 and distance<15:
   charge=18;	
   total=total+charge*distance
else:
   charge=15;	
   total=total+charge*distance
    
print ("Total charge:",total)

File: Assignment4/Assignment4.ts
var readline = require('readline');
var rl = readline.createInterface({
	input: process.stdin,
	output: process.stdout
  });
rl.question("Enter distance: ",(distance: number) => { 
var charge=0;
var total=0;  
	if(distance<=0){
		console.log('Invalid distance');
	}
	else if(distance>0 && distance<2){
	    charge=50;	
	    total=total+charge;
	}
    else if(distance>=2 && distance<5){
		charge=20;	
	    total=total+charge*distance;
	}
    else if(distance>=5 && distance<15){
		charge=18;	
	    total=total+charge*distance;
	}
    else if(distance>=15){
		charge=15;	
	    total=total+charge*distance;	
	}
        console.log(`Total charge is: ${total}`);

rl.close();
});